src/main/java/de/torui/coflsky/handlers/EventRegistry.java

```diff
@@ -236,7 +245,6 @@ public void onGuiOpen(GuiOpenEvent event) {
 	@SubscribeEvent(priority = EventPriority.HIGHEST)
 	public void onItemTooltipEvent(ItemTooltipEvent event) {
 		if (!config.extendedtooltips) return;
-		if(descriptionHandler == null) return;
```

Comment:
why was this removed?

Reply:
I didn't purposefully remove it may have been an accident or I made the fork before that line existed